### ISSUE TITLE
generatorの出力とembedding元との距離を損失に加える

### ISSUE THREAD

**Issue by KateSawada** [KateSawada/conditional-musegan-pytorch#29]
configから選択できるように．
Binary Cross Entropy, L1, L2あたりが考えられる．

**Comment by KateSawada** [comment]
gradient penaltyというものを使っていたらしいが，よくわからんので一旦省略することにする．

https://qiita.com/triwave33/items/5c95db572b0e4d0df4f0
